src/cljs/rems/actions/assign_external_id.cljs

```diff
@@ -2,13 +2,12 @@
   (:require [clojure.string :as str]
             [re-frame.core :as rf]
             [rems.actions.components :refer [action-button action-form-view button-wrapper command!]]
-            [rems.atoms :refer [textarea]]
             [rems.text :refer [text]]))
 
 (rf/reg-event-fx
  ::open-form
- (fn [{:keys [db]} _]
-   {:db (assoc db ::external-id "")}))
+ (fn [{:keys [db]} [_ initial-id]]
```

Comment:
We could also fetch the initial-id from the app-db, but so far the design has been to isolate these actions from the application. Cf. rems.actions.change-resources.